plugins/identity/app/services/service_layer/identity_service.rb

```diff
@@ -9,6 +9,9 @@ class IdentityService < Core::ServiceLayer::Service
     include IdentityServices::Group
     include IdentityServices::Role
     include IdentityServices::RoleAssignment
+    include IdentityServices::AWS::Ec2Creds
```

Comment:
I think you didn't get my point right yesterday. Sorry... Everything related to `Openstack Identity Service` should go to the identity service layer. And if there is any method missing from the identity api you should then do it in the identity service layer. You can fetch the identity service layer from any controller in your plugin.

These AWS modules are very specific and just for your plugin. Please move this modules back to the email_service layer. If it is not clear give me call. Thx!!

Reply:
moved back as before.